scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -283,11 +289,9 @@ task GetBQTableLastModifiedDatetime {
             gcloud auth activate-service-account --key-file=local.service_account.json
         fi
 
-        # bq needs the project name to be separate by a colon
-        DATASET_TABLE_COLON=$(echo ~{dataset_table} | sed 's/\./:/')
-
-        LASTMODIFIED=$(bq show  ~{"--project_id " + query_project} --location=US --format=json ${DATASET_TABLE_COLON} | python3 -c "import sys, json; print(json.load(sys.stdin)['lastModifiedTime']);")
+        echo "project_id = ~{data_project}" > ~/.bigqueryrc
 
+        LASTMODIFIED=$(bq --location=US --project_id=~{data_project} --format=json show ~{dataset}.~{dataset_table} | python3 -c "import sys, json; print(json.load(sys.stdin)['lastModifiedTime']);")
```

Comment:
I think this will also fail for us too since `query_project` is not being provided. `data_project` != `query_project` for our use case.